docs/functionalityValidation.md

```diff
@@ -0,0 +1,16 @@
+# Functionality Validation
+
+## Completed task
+
+A functionnality (or development task) is said to be finished if it meets all these conditions:
+* At least one code review has been done.
+* Integration tests have been performed.
+* The new development has been integrated into git.
+* The functionality has been tested and validated by the Product Owner.
```

Comment:
Could you move test by PO and documentation up, I think at 2nd for test and doc at first place